source/deploy/desktop-app.rst

```diff
@@ -3,7 +3,7 @@ Desktop App Deployment Guide
 
 Mattermost desktop applications are available for Windows, macOS, and Linux operating systems.
 
-You can `download the apps directly from our download page <https://mattermost.com/download/#mattermostApps>`__ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`__ for help during setup and for troubleshooting tips.
+You can `download the apps directly from our download page <https://mattermost.com/download/#>`__ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`__ for help during setup and for troubleshooting tips.
```

Comment:
Curious as to why the "mattermostApps" part was removed, as it appears to land in the correct section of the correct page.

Reply:
Well, when we hit that URL it resolves back to # at the end. And if you try again with this current URL containing #, then it still points to the same section of the page. However, if we use this URL for the first time, it just opens up the page. So, probably we can revert this. Thanks for pointing it out.